src/transport/SessionManager.cpp

```diff
@@ -389,7 +389,6 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
                     destination = &(multicastAddress.SetInterface(interfaceId));
                     if (mTransportMgr != nullptr)
                     {
-                        CHIP_TRACE_PREPARED_MESSAGE_SENT(destination, &tempBuf);
```

Comment:
Shouldn't we still trace that a message was sent, even if we don't include the data?

Reply:
I was thinking if we do want something like "we sent the encrypted bit" that should be more of a trace message (we have trace instant ... which does not work with existing infra but will work with the new one. Or if we find it important enough to not need to activate tracing, then maybe we should log.

We do logtrace the "send message" before encryption, this just removes the attempt to do the same again after encryption, hence my belief that it may not be needed. The only time I believe it may be useful is for MRP resend (when we only resend encrypted bits) however if we debug those, probably trace or MRP retry logs are better tools than passing around encrypted buffers for logging.